packages/connectivity/src/scp-cf/jwt/verify.ts

```diff
@@ -1,43 +1,11 @@
-import { createSecurityContext } from '@sap/xssec';
-import { ErrorWithCause } from '@sap-cloud-sdk/util';
-import { getXsuaaService } from '../environment-accessor';
-import type { JwtPayload } from '../jsonwebtoken-type';
-
-/**
- * Verifies the given JWT and returns the decoded payload.
- * @param jwt - JWT to be verified
- * @param options - Options to control certain aspects of JWT verification behavior.
- * @returns A Promise to the decoded and verified JWT.
- * @internal
- */
-export async function verifyJwt(
-  jwt: string,
-  options?: VerifyJwtOptions
-): Promise<JwtPayload> {
-  const disableCache = !{ ...defaultVerifyJwtOptions, ...options }
-    .cacheVerificationKeys;
-
-  const xsuaaService = getXsuaaService({ disableCache, jwt });
-
-  const { token } = await createSecurityContext(xsuaaService, {
-    jwt
-  }).catch(e => {
-    throw new ErrorWithCause('Failed to verify JWT.', e);
-  });
-
-  return token.payload;
-}
-
 /**
  * Options to control certain aspects of JWT verification behavior.
+ * @deprecated This property is no longer used as the incoming JWT is not verified during destination retrieval.
```

Comment:
[req] Generally a good practice to mention `@deprecated since vX.Y.Z`. Also I would say `The property has no effect as the incoming JWT is no longer verified during destination retrieval.`

Reply:
The interface deprecation should not say property